src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java

```diff
@@ -0,0 +1,62 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
+import org.fundacionjala.pivotal.pageobjects.dashboard.FormCreate;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+
+/**
+ * This class have common steps for cucumber tests.
+ */
+public class CommonSteps {
+    private static final Logger LOGGER = LogManager.getLogger("Common Steps");
+    private static Dashboard dashboard;
+    private static FormCreate formPage;
+
+    /**
+     * getter for dasboard.
+     * @return dashboardpage
+     */
+    public static Dashboard getDashboard() {
+        return dashboard;
+    }
+
+    /**
+     * getter for form page.
+     * @return formpage
+     */
+    public static FormCreate getFormPage() {
+        return formPage;
+    }
+
+    /**
+     * Precondition, user must be logged in.
+     *
+     * @param username inserted
+     * @param password inserted
+     */
+    @Given("^I log in as \"([^\"]*)\" \"([^\"]*)\"$")
+    public void iLogInAs(final String username, final String password) {
+        try {
+            dashboard = SignInPage.newCredentials(Environment.getInstance()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)